dl_controller:
  goal_features: [303.0,217.0,403.0,283.0,389.0,297.0]
